workshops/memory game/README.md

```diff
@@ -0,0 +1,376 @@
+
+```css
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: -apple-system, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
+    'Open Sans', 'Helvetica Neue', sans-serif;
+}
+body {
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+.board {
+  display: flex;
+  flex-wrap: wrap;
+  width: 400px;
+  height: 300px;
+}
+```
+
+Check your output.
+
+![CSS output](https://cloud-mv8zkaxw2.vercel.app/0screenshot_2020-10-31_133838.png)
+
+We completed the markup and styling of our project. Now flip over to `script.js` and let's start creating the game.
+
+## The JavaScript
+
+Create a DOM [event-listener](https://developer.mozilla.org/en-US/docs/Web/API/EventListener) for the event [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event). All our JavaScript code will be inside the event-listener, which will be excuted after the content loaded.
+
+```javascript
+document.addEventListener('DOMContentLoaded', () => {
+  // code goes here!
+})
+```
+
+Inside the event-listener, create an array for the cards which we use for the game. Add two of each cards for matching the cards. The public links for the images are provided in the code below. If you downloaded the images or using your own images, then add the relative path to the images. Also name them as your wish.
+
+```javascript
+document.addEventListener('DOMContentLoaded', () => {
+  const cardArray = [
+    {
+      name: '1',
+      img: 'https://cloud-5ystxzer7.vercel.app/11.png'
+    },
+    {
+      name: '2',
+      img: 'https://cloud-5ystxzer7.vercel.app/22.png'
+    },
+    {
+      name: '3',
+      img: 'https://cloud-5ystxzer7.vercel.app/33.png'
+    },
+    {
+      name: '4',
+      img: 'https://cloud-5ystxzer7.vercel.app/44.png'
+    },
+    {
+      name: '5',
+      img: 'https://cloud-5ystxzer7.vercel.app/55.png'
+    },
+    {
+      name: '6',
+      img: 'https://cloud-5ystxzer7.vercel.app/06.png'
+    },
+    {
+      name: '1',
+      img: 'https://cloud-5ystxzer7.vercel.app/11.png'
+    },
+    {
+      name: '2',
+      img: 'https://cloud-5ystxzer7.vercel.app/22.png'
+    },
+    {
+      name: '3',
+      img: 'https://cloud-5ystxzer7.vercel.app/33.png'
+    },
+    {
+      name: '4',
+      img: 'https://cloud-5ystxzer7.vercel.app/44.png'
+    },
+    {
+      name: '5',
+      img: 'https://cloud-5ystxzer7.vercel.app/55.png'
+    },
+    {
+      name: '6',
+      img: 'https://cloud-5ystxzer7.vercel.app/06.png'
+    }
+  ]
+})
+```
+
+Now we are going to creating our game board.
+
+## Game Board
+
+Pick up the `div` element with `class = "board"` using [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) and define it as `board`. Create a function `createBoard()` and loop over through elements in `cardArray` using `for` loop and add cards to our game board.
```

Comment:
Another place where you should split all of these steps up into step-by-step instructions with a code block.